conf/rails

```diff
@@ -16,20 +16,14 @@ LOGSDIR=/var/log/$APPNAME
 [ "$FAB_HTTP_PROXY" ] && export HTTP_PROXY=$FAB_HTTP_PROXY
 GEMOPTS="--no-rdoc --no-ri"
 
-gem update --system
+git clone https://github.com/rbenv/ruby-build.git /usr/local/src/ruby-build
```

Comment:
As a general rule, unless there's a good reason not too, shallow clones are best. I.e. `--depth 1`